lib/fog/compute/google/models/network.rb

```diff
@@ -36,12 +36,10 @@ def destroy(async = true)
           operation
         end
 
-        # Returns a ready API structure for insert_instance, used in insert_server request.
-        def get_as_interface_config(access_config = nil)
-          network_interface = { "network" => self_link }
-          network_interface["accessConfigs"] = [access_config] if access_config
-          network_interface
```

Comment:
After reading the code that uses this, I'm wondering if we should just delete the function and the one you replaced it with.